pygount/analysis.py

```diff
@@ -25,7 +25,7 @@
 import pygount.lexers
 import pygount.xmldialect
 from pygount.common import deprecated, mapped_repr
-from pygount.git_storage import GitStorage, git_remote_url_and_revision_if_any
+from pygount.git_storage import _GIT_REPO_REGEX, GitStorage, git_remote_url_and_revision_if_any
```

Comment:
Import only public names -> Rename to `GIT_REPO_REGEX`? Make a local constant here?